workers/unity/Packages/com.improbable.gdk.mobile/package.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "com.improbable.gdk.mobile",
     "displayName": "SpatialOS GDK Mobile Support",
-    "version": "0.0.2",
+    "version": "0.1.5",
```

Comment:
@jessicafalk pointed out that, as Mobile is in pre-alpha so the version of this should be lower than the rest of the GDK, which is in alpha. As a result I'm changing this from `0.1.5` to `0.0.3`